corpus_analysis/corpus.py

```diff
@@ -194,7 +194,7 @@ def __eq__(self, other):
         initializing a corpus.
         Presumes the documents to be sorted. (They get sorted by the initializer)
 
-        >>> from gender_analysis.corpus import Corpus
+        >>> from corpus_analysis.corpus import Corpus
         >>> from gender_analysis.common import TEST_DATA_PATH
```

Comment:
Should this path be updated? And/or, is there value in having `TEST_DATA_PATH` as a constant in `gender_analysis.common`?

Reply:
This would lead to a significant overhaul, as we import `TEST_DATA_PATH` from `gender_analysis.common` in many places. I'm comfortable leaving this as is: it may be the case that in the future we want a discrete testing path in `gender_analysis`.